lib/config/configuration.go

```diff
@@ -1017,29 +1015,33 @@ func Configure(clf *CommandLineFlags, cfg *service.Config) error {
 	// If application name was specified on command line, add to file
 	// configuration where it will be validated.
 	if clf.AppName != "" {
-		fileConf.Apps.Service.EnabledFlag = "yes"
+		cfg.Apps.Enabled = true
 
 		// Parse static and dynamic labels.
 		static, dynamic, err := parseLabels(clf.Labels)
 		if err != nil {
 			return trace.BadParameter("labels invalid: %v", err)
 		}
-		dynamicLabels := make([]CommandLabel, 0, len(dynamic))
+		dynamicLabels := make(services.CommandLabels)
```

Comment:
Actually, one suggestion is I think `parseLabels()` above already returns `services.CommandLabels` so you don't need to reconstruct it again, can just pass `dynamic` below.